evv4esm/extensions/tsc.py

```diff
@@ -163,18 +163,19 @@ def main(args):
         args.test_case += '1'
         args.ref_case += '2'
 
-    file_search_glob = '{d}/*cam_????.h0.0001-01-01-?????.nc.{s}'
+    file_search_glob_ref = '{d}/*eam_????.h0.0001-01-01-?????.nc.{s}'
+    file_search_glob = '{d}/*eam_????.h0.0001-01-01-?????.nc.{s}'
```

Comment:
Might want to explicitly call this one `file_search_glob_test` -- took me a minute to figure out what it was for. Right now, it's also the same as `file_search_glob_ref`?